gelinas/coding-challenge
# create django REST API for referral system

Features:
1) add `/admin/` endpoint for Django superuser to view database
2) create `Page` and `Referrer` models to track referrals
3) create `/api/referral/test` endpoint to verify server status
4) create `/api/referral/page` endpoint to verify affiliate status for given page
5) create `/api/referral/credit` endpoint to grant affiliates credit for their backlinks
6) seed database with Tim's webpage and three affiliates
7) create test suite that covers both models and all three endpoints

TODO:
1) Configure for Heroku Deployment
2) Document architecture and endpoints in README
3) Implement signed token authorization for viewing affiliate status and granting affiliate credit
